packages/teams-js/src/public/stageView.ts

```diff
@@ -53,11 +54,24 @@ export namespace stageView {
     return new Promise(resolve => {
       ensureInitialized(FrameContexts.content);
 
+      if (!isSupported()) {
+        throw errorNotSupportedOnPlatform;
+      }
+
       if (!stageViewParams) {
         throw new Error('[stageView.open] Stage view params cannot be null');
       }
 
       resolve(sendAndHandleSdkError('stageView.open', stageViewParams));
     });
   }
+
+  /**
+   * Checks if stageView capability is supported by the host
```

Comment:
While you're here, can you add @beta tags to this doc comment and to the top of the namespace please? (I assume that this capability is still under development. If not, then we should not add @beta and instead should unhide the doc comments for `open`.)

Reply:
Added beta tag to the stageView namespace to follow convention of other APIs in beta